app/assets/javascripts/controllers/live_metrics/util/metrics-parse-url-factroy.js

```diff
@@ -1,13 +1,43 @@
 angular.module('miq.util').factory('metricsParseUrlFactory', function() {
+  var re = /([^&=]+)=?([^&]*)/g;
+  var decodeRE = /\+/g;  // Regex for replacing addition symbol with a space
+  var decode = function(str) {
+    return decodeURIComponent( str.replace(decodeRE, ' ') );
+  };
+
+  var parseParams = function(query) {
```

Comment:
@simon3z @himdel 
Ahh.. we will have to use this hack, Can't find a better way that will work on older browsers :-(